common/templates/options.php

```diff
@@ -109,6 +108,26 @@
                 </p>
                 </td>
             </tr>
+            <tr valign="top">
+                <th scope="row"><?php _e('Product reviews', 'webwinkelkeur'); ?></th>
+                <td>
+                    <fieldset>
+                        <label>
+                            <input type="checkbox" name="<?= $plugin->getOptionName('product_reviews');?>" value="1" <?php if ($config['product_reviews']) {
+                                echo 'checked ';
+                            }?> />
+                            <?= esc_html(
+                                __('Import product reviews to WooCommerce.')
+                            ); ?>
+                            <p class="description">
+                                <?= esc_html(
+                                    __(sprintf('Automatically display product reviews collected using %s on your WooCommerce shop.', $plugin->getName()))
+                                ); ?>
```

Comment:
You need to apply `__` first and then `sprintf`.  This way it is not translatable.

Reply:
You need to add the `'webwinkelkeur'` parameter to `__` like it is done everywhere else.  This is the translation domain.